parser/testdata/workflows/applicationrequest-issue69.json

```diff
@@ -5,7 +5,7 @@
   "description": "Determine if applicant request is valid",
   "start": "CheckApplication",
   "specVersion": "0.8",
-  "auth": "./testdata/workflows/urifiles/auth.json",
+  "auth": "file://testdata/workflows/urifiles/auth.json",
```

Comment:
can we have one without _file://_ to make sure it will work as before?

Reply:
I mean, one test using the file without the protocol, as it was before, to make sure it still being unmarshaled correctly.